README.md

```diff
@@ -191,3 +192,21 @@ be organised in an appropriate sub-folder there.
   device you need BYOD & 2FA set up.
 * A publish user with your email address with access to the organisation(s) you
   want to modify.
+
+### DfE Sign-in
+
+To use it locally (and in qa) you will need a DfE Sign-in account.
```

Comment:
maybe reword it to state along the veins of `mandatory requirements` 
ie
to successfully login to system
1.  need to create an account on dfe
2.  assign organisation/provider or via god user

Otherwise you stuck

Reply:
Thanks, much clearer now!